src/FoneDynamics/Rest/V2/Messaging/MessageResource.cs

```diff
@@ -359,6 +359,7 @@ protected MessageResource()
         /// Gets a message against the account by its MessageSid.
         /// </summary>
         /// <param name="messageSid">The MessageSid of the message to retrieve.</param>
+        /// <param name="propertySid">The PropertySid of the property associated with the the message.</param>
```

Comment:
( see your account details page to obtain this no?)
or something similar?

Reply:
Instructions on PropertySid/TokenSid/etc are available on the README